.github/ISSUE_TEMPLATE/update_truth.md

```diff
@@ -26,9 +26,9 @@ assignees: ''
 - [ ] Select **engineer(s)** or **no engineer** required
 - [ ] Select **scientist(s)** or **no scientist** required
 
-### Projects and Milestone ###
-- [ ] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED CYCLE ASSIGNMENT** label
-- [ ] Select **Milestone** as the next official version or **Future Versions**
+### Milestone and Projects ###
+- [ ] Select **Milestone** as the next official version or **Backlog of Development Ideas**
+- [ ] For the next official version, select the **METplus-Wrappers-X.Y.Z Development** project
```

Comment:
If the truth data is being updated for an existing release, then the coordinated release support project should be selected instead. 
```suggestion
- [ ] If updating truth data for the `develop` branch, select the **METplus-Wrappers-X.Y.Z Development** project OR if updating truth for a `main_vX.Y` branch, select the **Coordinated METplus-X.Y Support** project
```